tests/integration/docusaurus/connecting_to_your_data/how_to_connect_to_sqlite_data.py

```diff
@@ -0,0 +1,80 @@
+import pathlib
+import sqlite3
+
+
+def get_sqlite_database_path() -> pathlib.Path:
```

Comment:
@TrangPham Could you please put a "usage example" docstring at the top -- other Python modules in the "fluent" directory have nice examples.  Thanks!